data/interfaces/default/info_search_results_list.html

```diff
@@ -246,10 +246,10 @@ <h4>Tracks</h4>
                             </div>
                         </div>
                     </div>
+                    % if _session['user_group'] == 'admin':
+                    <span class="overlay-refresh-image" title="Refresh image"><i class="fa fa-refresh refresh_pms_image"></i></span>
+                    % endif
                 </div>
-                % if _session['user_group'] == 'admin':
-                <span class="overlay-refresh-image" title="Refresh image"><i class="fa fa-refresh refresh_pms_image"></i></span>
-                % endif
                 <div class="item-children-instance-text-wrapper album-item">
```

Comment:
Could you also change `album-item` to `cover-item` while you're fixing this file?

There are three `album-item` in this file.